example.env

```diff
@@ -1 +1 @@
-REACT_APP_APIKEY='fd1269ab93ffa172ea8a1e4f11d51867'
```

Comment:
it seems that you have pushed your API key in one of the commits

Event if you delete it, the API key will still be accessible through the commit history. 

Please go to the open weather dashboard and delete this API key and make a new one